buildlogger.proto

```diff
@@ -42,3 +42,16 @@ message LogEndInfo {
   int32 exit_code = 1;
   google.protobuf.Timestamp completed_at = 2;
 }
+
+message BuildloggerResponse {
+  string log_id = 1;
+  bool success = 2;
+}
+
+
+service Buildlogger {
+  rpc CreateLog(LogInfo) returns (BuildloggerResponse);
+  rpc AppendLogLines(LogLines) returns (BuildloggerResponse);
```

Comment:
this should probably be "repeated" for the log lines. because it's a single line. ( you could change the name to be signular, but I think we should encourage people to batch in some way.

Reply:
No, LogLines is multiple lines (there is LogLine and LogLines)